code/game/objects/items/weapons/tools.dm

```diff
@@ -453,6 +453,9 @@
 		var/datum/organ/internal/eyes/E = H.internal_organs_by_name["eyes"]
 		if(!E)
 			return
+		if(E.welding_proof)
+			usr.simple_message("<span class='notice'>Your eyelenses darken to accommodate for the welder's glow</span>")
```

Comment:
If it's not too much of a bother, please change the 5 or so instances of usr to user in this proc